### PR TITLE
Re-add and update audit logging documentation

### PR DESCRIPTION
**Pull request by embik**
This adds (back) a documentation page on Kubernetes audit logging, a feature supported by KKP to log and review the operations issued against the Kubernetes API. This page exists in KKP docs version 2.16, but isn't in 2.18 anymore.

This was (in error?) removed in https://github.com/kubermatic/docs/pull/514/files#diff-c55924e2625c20ff6cd72e2151ba5b49883422042bf52e19a48c0b60fc029970, as far as I read that PR. I think this was by mistake because the feature is barely documented at the moment and needs clarification for users. As of now, it's basically a checkbox in the UI, but that's it.

fixes kubermatic/edgematic#75

### CODE REVIEW

**Comment by embik** [comment]
xref https://github.com/kubermatic/edgematic/issues/75

**Comment by embik** [comment]
fixes https://github.com/kubermatic/edgematic/issues/75

**Review by mlavacca — COMMENTED** [review]
The PR looks good, I only left one little comment.

File: .gitignore [hunk 1]
```diff
@@ -8,3 +8,6 @@ tutorials-tests/cypress/fixtures
 
 # IDE specific files
 .idea/
+
+# macOS files
+.DS_Store
```
[the remark this answers is not in the document]
Comment (embik) [reply .gitignore:13]: That's a very good point. Let me remove it from the PR.

**Pull request merged**